src/toil/worker.py

```diff
@@ -133,9 +134,7 @@ def badWorker():
 
     setLogLevel(config.logLevel)
 
-    tempRootDir = config.workDir
-    if tempRootDir is not None and not os.path.exists(tempRootDir):
-        raise RuntimeError("The temporary directory specified by workDir: %s does not exist" % tempRootDir)
+    tempRootDir = getToilWorkflowDir(config.workflowID, config.workDir)
```

Comment:
Should we rename tempRootDir or would that be too invasive, i.e. affect many lines in the code?